modifytags/modifytags.go

```diff
@@ -0,0 +1,367 @@
+package modifytags
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"go/ast"
+	"go/token"
+	"sort"
+	"strconv"
+	"strings"
+	"unicode"
+
+	"github.com/fatih/camelcase"
+	"github.com/fatih/structtag"
+)
+
+// A Mutation defines how the input is modified.
+type Mutation struct {
+	Remove        []string            // Remove tags for the comma separated list of keys.
```

Comment:
Is the list comma separated anymore? It is now a slice, no?
Also, the tense of these comments seems a bit off. They should describe what the field holds, not be framed as an instruction ("Remove tags").

How about just `// tags to remove`. No need for a capital letter or punctuation when the comment is not a full sentence. Indeed, punctuation should be avoided in this case.

Ditto for 'Add' below.